libuuu/zip.cpp

```diff
@@ -286,7 +286,9 @@ int	Zip_file_Info::decompress(Zip *pZip, shared_ptr<FileBuffer>p)
 		switch (ret) {
 		case Z_NEED_DICT:
 			ret = Z_DATA_ERROR;     /* and fall through */
+			[[fallthrough]];
```

Comment:
Fall through is C++ 17 standard,   some one still use vs2017,  I don't think vs2017 have good support for C++17

Reply:
Done!